snuba/datasets/outcomes.py

```diff
@@ -60,6 +64,24 @@ def process_message(self, value, metadata=None) -> Optional[ProcessedMessage]:
         return ProcessedMessage(action=ProcessorAction.INSERT, data=[message],)
 
 
+class OutcomesQueryStorageSelector(QueryStorageSelector):
```

Comment:
This is one of the missing pieces to expose a raw datamodel from matviews and transparently run the query on either the materialized storage or the raw table storage.

Reply:
If I'm reading this correctly, this is just here to facilitate the eventual filling in of this class (since right now it could just be a single table storage?)

For reviewing's sake, I think would have been clearer to implement this using the single storage selector and noted the `TODO` at it's instantiation point — there's not a ton of code here so it's not really a big deal but adding this class makes the overall change larger than it needs to be.

Reply:
it is correct. I will turn it into a single table storage